process/block/baseProcess_test.go

```diff
@@ -1549,3 +1550,278 @@ func TestBaseProcessor_updateState(t *testing.T) {
 	assert.Equal(t, []byte("rootHash"), pruneRootHash)
 	assert.Equal(t, []byte("rootHash"), cancelPruneRootHash)
 }
+
+func TestBaseProcessor_ProcessScheduledBlockShouldFail(t *testing.T) {
+	t.Parallel()
+
+	t.Run("execute all scheduled txs fail", func(t *testing.T) {
+		arguments := CreateMockArguments(createComponentHolderMocks())
```

Comment:
you can add t.Parallel on all inner tests. It will run all the inner tests in parallel and if you keep t.Parallel at the "parent" level (say, L1555) it will run all inner tests in parallel with other inner tests